scripts/docker-build

```diff
@@ -42,8 +42,7 @@ if [[ "${1:-}" == "--help" ]]; then
   exit 0
 fi
 
```

Comment:
These two lines are moved into a docker build "pre" script, to avoid future situations where code is added to one but not the other docker build.